k8s/docs/values-prod.yaml

```
@@ -29,7 +29,7 @@ autoscaling:
   maxReplicas: 3
   targetCPUUtilizationPercentage: 70
 nodeSelector: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)